apptax/taxonomie/models.py

```diff
@@ -1,20 +1,70 @@
 import os.path
 
 from flask import current_app
-from flask_sqlalchemy import SQLAlchemy
+
+
+from packaging import version
+import flask_sqlalchemy
+
+if version.parse(flask_sqlalchemy.__version__) >= version.parse("3"):
+    from flask_sqlalchemy.query import Query
+else:
+    from flask_sqlalchemy import BaseQuery as Query
+
 from sqlalchemy import ForeignKey, select, func
 
 from sqlalchemy.ext.hybrid import hybrid_property
 from sqlalchemy.schema import FetchedValue
-from sqlalchemy.orm import deferred
+from sqlalchemy.orm import deferred, raiseload, joinedload
 
-from flask_sqlalchemy import BaseQuery
 from utils_flask_sqla.serializers import serializable
 from ref_geo.models import LAreas
 
 from . import db
 
 
+class TaxrefQuery(Query):
+    def joined_load(self, fields=None):
+        qs = self
+
+        query_option = [raiseload("*")]
+        if fields:
+            for f in fields:
+                if f in Taxref.__mapper__.relationships:
+                    query_option.append(joinedload(getattr(Taxref, f)))
+        else:
+            for r in Taxref.__mapper__.relationships:
+                query_option.append(joinedload(getattr(Taxref, r.key)))
+        qs = qs.options(*tuple(query_option))
+
+        return qs
+
+    def id_listes_filters(self, id_liste):
```

Comment:
```suggestion
    def filter_by_lists(self, id_liste: list):
```
Juste une suggestion de renommage